src/_includes/section.edde.njk

```diff
@@ -1,40 +1,39 @@
 {% for item in collections.sections %}
   {% if item.data.type === 'edde' %}
-    <div id="edde" class="bg-edde-pattern xl:p-40 md:p-16 p-8 py-32">
-      <div class="flex flex-col text-center pb-4 xl:pb-16">
-        <h2 class="xs:text-md tracking-widest text-purple-650 mt-20">{{ item.data.title }}</h2>
+    <div id="edde" class="bg-edde-pattern py-16 px-8">
+      {# Title, Subtitle, First P #}
+      <div class="grid grid-cols-1 gap-2 text-center lg:px-36">
+        <h2 class="tracking-widest text-purple-650 mt-20">{{ item.data.title }}</h2>
         <div class="flex justify-center items-center flex-row flex-wrap">
-          <p class="xs:text-2xl sm:text-4xl font-bold p-2">
+          <p class="text-2xl font-medium tracking-wider p-2 sm:text-3xl">
```

Comment:
Let's group our responsive styles from mobile to desktop like this: "text-2xl sm:text-3xl ..."